packages/contentstack-utilities/package.json

```diff
@@ -32,7 +32,7 @@
   "author": "contentstack",
   "license": "MIT",
   "dependencies": {
-    "@contentstack/management": "~1.17.0",
+    "@contentstack/management": "~1.18.0",
```

Comment:
@sunil-lakshman , Please update the management SDK version anywhere the cli is used, such as the contentstack root folder or import.